addons/web/static/src/views/calendar/calendar_model.js

```diff
@@ -411,7 +411,11 @@ export class CalendarModel extends Model {
         // Compute the domain
         const domain = [];
         for (const field in authorizedValues) {
-            domain.push([field, "in", authorizedValues[field]]);
+            if (field === "partner_ids") {
+                const vals = authorizedValues[field];
+                domain.push("|", [field, "in", vals], ["partner_id", "in", vals]);
+            }
+            else domain.push([field, "in", authorizedValues[field]]);
```

Comment:
It does not seem to me that it concerns the base view, which should be as generic as possible.

Instead override the `computeFiltersDomain` method in the `attendee_calendar_model.js`.

In order to facilitate this override, you should move the first part of this method in a new one:
```js
    /**
     * List authorized values for every field
     * fields with an active "all" filter are skipped.
     **/
    getAuthorizedValuesForFilters(data) {
        const authorizedValues = {};
        const avoidValues = {};
        for (const [fieldName, filterSection] of Object.entries(data.filterSections)) {
            // Skip "all" filters because they do not affect the domain
            const filterAll = filterSection.filters.find((f) => f.type === "all");
            if (!(filterAll && filterAll.active)) {
                const filterSectionInfo = this.meta.filtersInfo[fieldName];
                // Loop over subfilters to complete authorizedValues
                for (const filter of filterSection.filters) {
                    if (filterSectionInfo.writeResModel) {
                        if (!authorizedValues[fieldName]) {
                            authorizedValues[fieldName] = [];
                        }
                        if (filter.active) {
                            authorizedValues[fieldName].push(filter.value);
                        }
                    } else {
                        if (!filter.active) {
                            if (!avoidValues[fieldName]) {
                                avoidValues[fieldName] = [];
                            }
                            avoidValues[fieldName].push(filter.value);
                        }
                    }
                }
            }
        }
        return [authorizedValues, avoidValues];
    }
```

Usage:
```js
    computeFiltersDomain(data) {
        const [authorizedValues, avoidValues] = this.getAuthorizedValuesForFilters(data);
        const domain = [];
        for (const field in authorizedValues) {
            domain.push([field, "in", authorizedValues[field]]);
        }
        for (const field in avoidValues) {
            if (avoidValues[field].length > 0) {
                domain.push([field, "not in", avoidValues[field]]);
            }
        }
        return domain;
    }
```